sdk/core/src/response.rs

```diff
@@ -13,6 +13,7 @@ pub(crate) struct ResponseBuilder {
 }
 
 impl ResponseBuilder {
+    #[allow(dead_code)]
```

Comment:
This maybe points to that this module should only be built if the reqwest feature is turned on.

Reply:
I updated to use cfg.